src/back/space/src/api/introspect.cpp

```diff
@@ -46,8 +46,8 @@ std::string Introspect::HandleRequestThrow(
 		auto userId = params[headers::kUserId].As<std::string>();
 		model::Space space = _s.GetByKeyIfAdmin(spaceKey, userId);
 		const std::string spaceIdStr = boost::uuids::to_string(space.id);
-		const std::string role = "admin";
-		const std::string token = _s.CreateToken(spaceIdStr, space.key, userId, role);
+		const int roleId = 3; // hardcoded admin roleId (3)
```

Comment:
А почему тут захардкожен админ? Получается только админ сможет зайти в пространство.
```suggestion
		const auto [space, roleId] = _s.GetSpaceAndRoleId(spaceKey, userId);
```

Reply:
Переписал, готово, влил.